### DISCUSSION TITLE
withdrawExitFaster method gives error

### DISCUSSION THREAD

**Discussion by m-a-x-i-m-u-s** [0xPolygon/matic.js#329] [Q&A]
When calling withdrawExitFaster with a transaction that has been checkpointed, it gives an error with code -32000 and  message: ''nonce too low". 
PFA, a screenshot of the code and the error
- code
![image](https://user-images.githubusercontent.com/85619385/162342381-ede62098-9e5c-40b1-836b-706d5a6a0382.png)
- error
![image](https://user-images.githubusercontent.com/85619385/162342529-362db7aa-824f-4026-b470-9d7d67e3fd0a.png)

**Comment by ujjwalgupta94** [comment]
It might be that some transaction are pending or something for that user, internally matic.js calculates the nonce and use to send the transaction. But you can also provide a nonce value from the api.

```
withdrawExitFaster(txHash,{
 nonce: 5
})
```

I hope this helps.
